astronomer/providers/google/cloud/triggers/bigquery.py

```diff
@@ -68,6 +68,7 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 if response_from_hook == "success":
                     yield TriggerEvent(
                         {
+                            "jobId": self.job_id,
```

Comment:
do we need to modify the unit tests to account for this additional parameter?

Reply:
Yes, accommodated the required change.